py/openage/codegen/__main__.py

```diff
@@ -89,7 +89,9 @@ def main():
                           "this causes codegen to abort with an error message "
                           "if the target cache has changed."))
 
-    ap.add_argument("--verbose", "-v", action='count', default=0)
+    ap.add_argument("--verbose", "-v", action='count', default=0,
+                    help=("print more stuff for debugging,"
+                          "repeat for even more output."))
```

Comment:
Did you really edit this file just to write a help text that explains how `--verbose` works?